sqlcipher_flutter_libs/lib/sqlcipher_flutter_libs.dart

```diff
@@ -0,0 +1,56 @@
+/// This library contains a workaround neccessary to open dynamic libraries on
+/// old Android versions (6.0.1).
+///
+/// The purpose of the `sqlcipher_flutter_libs` package is to provide `sqlite3`
+/// native libraries when building Flutter apps for Android and iOS.
+library sqlcipher_flutter_libs;
+
+import 'dart:ffi';
+import 'dart:io';
+import 'dart:math';
+
+import 'package:flutter/services.dart';
+
+const _platform = MethodChannel('sqlcipher_flutter_libs');
+
+/// Workaround to open sqlite3 on old Android versions.
+///
+/// On old Android versions, this method can help if you're having issues
+/// opening sqlite3 (e.g. if you're seeing crashes about `libsqlcipher.so` not
+/// being available). To be safe, call this method before using apis from
+/// `package:sqlite3` or `package:moor/ffi.dart`.
+///
+/// Big thanks to [@knaeckeKami](https://github.com/knaeckeKami) for finding
+/// this workaround!!
+Future<void> applyWorkaroundToOpenSqlite3OnOldAndroidVersions() async {
```

Comment:
We might want to call this `applyWorkaroundToOpenSqlCipherOnOldAndroidVersions`, but either way should be fine.

Reply:
i didnt call it  because every programmer may have a diffrent usecase to call it

in main.dart or in splash screen or before using database ...

Reply:
Sorry, I meant that we might want to rename this method to `applyWorkaroundToOpenSqlCipherOnOldAndroidVersions`, since the workaround we're applying is for sqlcipher instead of sqlite3.

Reply:
ohh sorry about that i rename it